core/src/components/input/input.ionic.scss

```diff
@@ -7,6 +7,8 @@
 // --------------------------------------------------
 
 :host {
+  // TODO(ROU-5477): use design token here when it is added
+  --color: #121212;
```

Comment:
Was it verified that it's going to be added? Else we might need to reach out to the designers to verify that this hex is the correct value.

Reply:
Yes I have verified that the design tokens for state have yet to be added. I actually asked about this specific hex before because I ran into it in other components. They are planned in ROU-4870 which is linked as a blocker to the ticket I added here.